File: src/m/modules/ElementConnectivity.m
function elementconnectivity = ElementConnectivity(elements,nodeconnectivity);
%ELEMENTCONNECTIVITY - Build element connectivity using node connectivity and elements
%
%   Usage:
%      elementconnectivity = ElementConnectivity(elements,nodeconnectivity);

% Check usage
if nargin~=2
	help ElementConnectivity
	error('Wrong usage (see above)');
end

% Call mex module
elementconnectivity = ElementConnectivity_matlab(elements,nodeconnectivity);

File: src/m/modules/ElementConnectivity.py
from ElementConnectivity_python import ElementConnectivity_python


def ElementConnectivity(elements, nodeconnectivity):
    """ELEMENTCONNECTIVITY - Build element connectivity using node connectivity and elements

    Usage:
        elementconnectivity = ElementConnectivity(elements, nodeconnectivity)
    """
    
    # Call Python module
    elementconnectivity = ElementConnectivity_python(elements, nodeconnectivity)

    return elementconnectivity[0] # NOTE: Value returned from wrapper function is a tuple, the first element of which being the result we actually want
